triaging.rst

```diff
@@ -35,17 +35,23 @@ Responsibilities include:
     - Good first issue
     - Other categorizations
 
-As triagers gain experience, they may have some intuition of when a PR should
-be closed. Triagers can recommend closing a PR, but the final decision must be
-made by a core developer. By having triagers and core developers work together,
+Although triagers have the power to close PRs, they should generally not do so
+without first consulting a core developer. By having triagers and core developers work together,
 the author receives a careful consideration of their PR. This encourages future
 contributions, regardless of whether their PR is accepted or closed.
 
-Triagers can make use of the ``invalid`` and ``stale`` labels to suggest that a
+Nonetheless, triagers should feel free to close a PR if they judge that the
+chance of the PR being merged is exceedingly low. This can include PRs that
+make solely cosmetic changes to a module, PRs that propose changes to
+deprecated modules, and/or PRs that self-evidently lie far below the quality
+threshhold required for a change to be made to code in the Python GitHub
+repositories.
```

Comment:
I wonder if we can reword this so that it's clearer what is meant. Often we get a PR with no unit test, and that makes it far below the threshold for making the change, but we don't close it, we ask for a unit test.

Reply:
I was thinking of PRs like, for example, the recently closed 32408 (deliberately not linking to it directly). That PR isn't a spammy PR — the author clearly has the best of intentions, has signed the CLA, and has even opened a BPO issue. But, with the best will in the world, that PR had a 0% chance of ever being merged.

How do we describe a PR like that?

Reply:
I think the issue here is not so much the quality of the PR as much as the quality of the whole idea (the quality of the bpo). We have a lot of open issues with bad enhancement suggestion. How to weed them out is one of the questions I hope we can discuss at the language summit.

Reply:
Yes. It's easy to say "this is not a bug". It's much more difficult to say "I'm sorry, you may think that this is a good idea, but it just isn't, and we're never realistically going to implement it".

Reply:
Agreed. Maybe we leave such things out of the dev guide and just focus on the really obvious ones.

Reply:
Sure. I'll make the change.

Reply:
Would it be worth adding something like "if in doubt, leave it to a core dev"?

Reply:
How's it look after f2166e7?